components/home/Header/styled.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components";
+
+interface ContainerProps {}
+export const Container = styled.div<ContainerProps>`
+  display: flex;
+  flex-direction: column;
+`;
+
+interface NavWrapperProps {}
+export const NavWrapper = styled.div<NavWrapperProps>`
+  background-color: #ff652e;
+  padding: 20px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+interface LogoProps {}
+export const Logo = styled.div<LogoProps>`
+  font-family: "PT Mono", monospace;
+  text-align: center;
+
+  h1 {
```

Comment:
should be sub components

Reply:
Do you mean I should create separate component for this, instead of nested tag based styling?